iceoryx_dds_gateway/test/moduletests/test_iox2dds.cpp

```diff
@@ -0,0 +1,337 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
I really like your tests and your style to write them - clean and understandable!